src/components/Layouts/Primary/TopBar/Items/AddMenu/AddMenu.tsx

```diff
@@ -1,53 +1,171 @@
 import React, { ReactElement, useState } from 'react';
-import { Box, IconButton, makeStyles, Theme, Menu } from '@material-ui/core';
+import {
+  Box,
+  Dialog,
+  IconButton,
+  ListItemText,
+  Menu,
+  styled,
+} from '@material-ui/core';
 import AddIcon from '@material-ui/icons/Add';
+import PersonIcon from '@material-ui/icons/Person';
+import PeopleIcon from '@material-ui/icons/People';
+import CardGiftcardIcon from '@material-ui/icons/CardGiftcard';
+import ListIcon from '@material-ui/icons/FormatListBulleted';
+import EditIcon from '@material-ui/icons/Edit';
 import { useTranslation } from 'react-i18next';
+import { useApp } from '../../../../../App';
+import CreateContact from './Items/CreateContact/CreateContact';
 
-const useStyles = makeStyles((theme: Theme) => ({
-  link: {
-    textTransform: 'none',
-    color: 'rgba(255,255,255,0.75)',
-    transition: 'color 0.2s ease-in-out',
-    '&:hover': {
-      color: 'rgba(255,255,255,1)',
-    },
+const HoverAddIcon = styled(AddIcon)(() => ({
+  textTransform: 'none',
+  color: 'rgba(255,255,255,0.75)',
+  transition: 'color 0.2s ease-in-out',
+  '&:hover': {
+    color: 'rgba(255,255,255,1)',
```

Comment:
This is just `white` right? Can you use `theme.palette.common.white`? Is there a theme color for the 75% version? The `IconButton` doesn't have the styles you want?

Reply:
Yeah, it's just white. I copied these from the original ```NotificationMenu``` that tatai setup. I could do 
```
 color: theme.palette.common.white,
  opacity: '0.75',
  transition: 'color 0.2s ease-in-out',
  '&:hover': {
    opacity: '1',
  },
```
what do you think of that instead?

Reply:
I like that better :)